tradesman/model_creation/synthetic_population/create_geo_crosswalk.py

```diff
@@ -0,0 +1,27 @@
+import csv
+import pandas as pd
+from aequilibrae import Project
+from os.path import join
+
+
+def create_geo_cross_walk(project: Project, file_folder: str):
+
+    qry = "SELECT zone_id FROM zones;"
+
+    zone_id = [i[0] for i in project.conn.execute(qry).fetchall()]
+
+    regions_and_pumas = [1 for i in list(range(len(zone_id)))]
+
+    create_lines = list(zip(zone_id, regions_and_pumas, regions_and_pumas))
+
+    df = pd.DataFrame(create_lines, columns=["TAZ", "PUMA", "REGION"])
+
+    df.to_csv(
```

Comment:
The WHOLE THING could also be substituted by the following:


```suggestion
    qry = "SELECT zone_id TAZ, 1 PUMA, 1, REGION FROM zones;"
    pd.read_sql(qry, project.conn).to_csv("data/geo_cross_walk.csv", index=False, quoting=csv.QUOTE_NONNUMERIC)
    df.to_csv(
```